src/reducers/notebook-reducer.js

```diff
@@ -117,47 +123,6 @@ const notebookReducer = (state = newNotebook(), action) => {
       return Object.assign({}, state, { mode })
     }
 
-    case 'CHANGE_SIDE_PANE_MODE': {
-      const { sidePaneMode } = action
-      return Object.assign({}, state, { sidePaneMode })
-    }
-
-    case 'CHANGE_SIDE_PANE_WIDTH': {
-      const width = state.sidePaneWidth + action.widthShift
-      return Object.assign({}, state, { sidePaneWidth: width })
-    }
-
-    case 'CHANGE_EDITOR_WIDTH': {
-      const width = state.editorWidth + action.widthShift
-      return Object.assign({}, state, { editorWidth: width })
-    }
-
-    /* eslint-disable prefer-destructuring */
-
-    case 'INCREASE_EDITOR_WIDTH': {
```

Comment:
do you find these fcns / associated shortcuts unncessary or will they be re-implemented later?

Reply:
def need to delete